File: conference/registration-bc/web/src/main/java/com/microsoft/conference/registration/web/Models/OrderItemViewModel.cs
using Registration.ReadModel;

namespace Registration.Web.Models
{
    public class OrderItemViewModel
    {
        public Registration.ReadModel.SeatType SeatType { get; set; }
        public OrderLine OrderLine { get; set; }
        public int Quantity { get; set; }
        public int MaxSelectionQuantity { get; set; }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/controller/OrderItemViewModel.java
package com.microsoft.conference.registration.controller;

import com.microsoft.conference.registration.readmodel.service.OrderLineVO;
import com.microsoft.conference.registration.readmodel.service.SeatTypeVO;
import lombok.Data;

@Data
public class OrderItemViewModel {
    private SeatTypeVO seatTypeVO;
    private OrderLineVO orderLineVO;
    private int quantity;
    private int maxSelectionQuantity;
}